docs/modules/deployment/pages/core/getting-started.adoc

```diff
@@ -30,6 +30,9 @@ endif::[]
 * Internet access to download the installation packages.
 * A working DNS server, and a localhost and server name that resolve properly.
 * A system user with administrative permissions (`sudo`) to perform installation.
+* A SELinux policy that permits binding to the ICMP service.
```

Comment:
It's possible (though not a good security practice) to disable SELinux.   There also are some operating systems that do not include SELinux and don't require these steps done.
```suggestion
* A SELinux policy that permits binding to the ICMP service.
(If you are using an operating system that includes SELinux)
```